packages/core/test/test-util/test-services/v2/test-service/TestEntity.ts

```diff
@@ -361,4 +353,12 @@ export namespace TestEntity {
     acc[field._fieldName] = field;
     return acc;
   }, {});
+
+  /**
+   * Returns a request builder to construct requests for operations on the `TestEntity` entity type.
+   * @returns A `TestEntity` request builder.
+   */
+export function requestBuilder(): TestEntityRequestBuilder {
```

Comment:
Is this a new design to swich from a static method `TestEntity.requestBuilder()` to a function `reqeustBuilder()`?
If yes.
- breaking change
    - new import needs to be adjusted
    - Multiple `requestBuilder()` of different entities might be used in a service, then aliases need to be created for naming conflict.
- generator needs to be adjusted.

Reply:
This was also for testing. I revert.